src/sentry/middleware/access_log.py

```diff
@@ -74,11 +74,9 @@ def _create_api_access_log(
             if resolver_match is not None
             else "Unknown"
         )
-        group = (
-            get_rate_limit_config(resolver_match.func.view_class).group
-            if resolver_match
-            else RateLimitConfig().group
-        )
+        func = getattr(resolver_match, "func", None)
+        view_class = getattr(func, "view_class", None) if func else None
+        group = get_rate_limit_config(view_class).group if view_class else RateLimitConfig().group
```

Comment:
This will maybe fix the errors being thrown but judging by how often this happens I don't think you will ever log the correct rate limit group

![image](https://user-images.githubusercontent.com/3169433/162815657-fc7468dd-e82e-4096-a222-e9a41dcd2911.png)

Reply:
Do you think I should revert then or merge this to fix the errors? This sounds like another case of "it works locally but not on prod".

Reply:
Revert https://github.com/getsentry/sentry/pull/33516 I think is the better option here

Reply:
yes the revert is better than this fix. Let's talk about a better fix together